service/arm_templates/main.json

```diff
@@ -152,9 +162,9 @@
         },
         "parameters": {
           "clusterID": {"value": "[parameters('clusterID')]"},
-          "apiLoadBalancerCidr": {"value": "[parameters('apiLoadBalancerCidr')]"},
+          "apiLoadBalancerCidr": {"value": "[reference('api_load_balancer_setup').outputs.loadBalancerIPAddress.value]"},
           "etcdLoadBalancerCidr": {"value": "[parameters('etcdLoadBalancerCidr')]"},
```

Comment:
Even though it's private, can the cidr be taken from the output for consistency reasons?

Reply:
The logical place from where to take the cidr would be from the network_setup and then pass it to the security_groups_setup template, but that would put us in a deadlock :(